packages/core-extensions/src/moonbase/ui/config/index.tsx

```diff
@@ -108,15 +109,39 @@ export default (require: WebpackRequireType) => {
   }
 
   return function ConfigPage() {
+    const TitleBarClasses = spacepack.findByCode("iconWrapper:", "children:")[0]
+      .exports;
     return (
       <>
+        <div
+          className={`${TitleBarClasses.children} ${Margins.marginBottom20}`}
+        >
+          <Text
+            className={TitleBarClasses.titleWrapper}
+            variant="heading-lg/semibold"
+            tag="h2"
+          >
+            Config
+          </Text>
+        </div>
+
         <FormItem title="Repositories">
           <FormText className={Margins.marginBottom4}>
             A list of remote repositories to display extensions from
           </FormText>
           <ArrayFormItem config="repositories" />
         </FormItem>
-        <FormDivider className={FormClasses.dividerDefault} />
```

Comment:
You killed the divider here, was that intended?

Reply:
Yup, FormSwitch gives a divider

Reply:
Yeah, but that's on the bottom.

Reply:
> Yeah, but that's on the bottom.

It showed up with a double divider for me when i tried, so /shrug